tutorial/write-custom-widget.md

```diff
@@ -235,7 +235,9 @@ We do this in the renderer because only the renderer knows the size of the canva
 
 ```go
 func (r *myWidgetRenderer) MinSize() fyne.Size {
-	return fyne.MeasureText(r.text.Text, r.text.TextSize, r.text.TextStyle)
+	// Measure the size of the text so we can calculate an indent.
+	ts := fyne.MeasureText(r.text.Text, r.text.TextSize, r.text.TextStyle)
+	return fyne.NewSize(ts.Width+20, ts.Height+20)
```

Comment:
I'm not sure about this magic 20, what is the reason?
If you want it to be padded from the outside consider using a multiple of `theme.Padding()` like other widgets (see https://developer.fyne.io/architecture/widgets)